icons/pypy.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PyPy icon</title><path d="M12.645.015a2.7 2.7 0 0 0-1.278.456c-.125.006-.247.026-.37.037-1.838.257-3.128 1.184-4.028 2.294-1.69 2.497-1.711 6.045-1.487 8.424a6.64 6.64 0 0 1 2.315-4.14l.057-.04a8.72 8.72 0 0 1 .435-.325c.028-.014.056-.027.084-.055.155-.098.295-.183.45-.281l.083-.043c.155-.085.31-.155.479-.225.028-.014.07-.029.097-.043.127-.056.269-.097.395-.139a2.186 2.186 0 0 0-.268-.24 1.823 1.823 0 0 0-.73-.266 2.37 2.37 0 0 0-.342-.04c-.023 0-.045-.006-.068-.007a4.648 4.648 0 0 0-.42.004h-.014a4.354 4.354 0 0 0-.818.123c.264-.11.527-.208.818-.264a3.78 3.78 0 0 1 .871-.111h.008c.087 0 .178.013.268.021.06.006.12.004.181.014l.004.002c.153.026.307.068.453.13.24.1.45.267.633.464.004 0 .007 0 .01-.002l.004.002a2.819 2.819 0 0 0 2.416.956l.254-.041h.041l.566-.116c1.853-.431 2.176-.506 3.123-1.033.4-.322.562-.549.631-.763.07-.217.055-.455.072-.797l-.002.01c-.001-.374-.281-.799-.695-1.22-.41-.417-.932-.824-1.365-1.196-.53-.374-.785-.718-1.031-.973a1.644 1.644 0 0 0-.678-.456l-.004-.002c-.004 0-.008 0-.012-.002a2.789 2.789 0 0 0-1.138-.117zm2.705.956c.11.101.205.2.359.309l.008.004.006.005c.424.366.962.78 1.4 1.226.438.446.795.93.797 1.466v.004l-.002.006c-.016.318.008.588-.088.885-.096.297-.31.588-.754.94l-.012.01-.011.008a9.766 9.766 0 0 1-.584.302c1.23.486 2.465 1.072 3.806 2.12v-.003c1.192.924 2.313 2.026 3.442 3.67C23.582 6.736 20.1 2.403 15.35.972zM8.533.983A11.619 11.619 0 0 0 3.71 3.88a11.635 11.635 0 0 0-3.426 8.183c.042 1.823.678 3.187 1.682 4.189 1.048 1.046 2.5 1.7 4.08 2.032 1.586.333 3.25.44 4.748.358a6.612 6.612 0 0 1-5.121-4.603l-.006-.017c-.014-.049-.03-.096-.043-.145-.06-.176-.222-.681-.402-1.806v-.006c-.3-2.348-.525-6.538 1.463-9.469l.004-.006.003-.006A6.817 6.817 0 0 1 8.533.983zm4.955 3.854c.123.01.238.034.342.066h.002c.208.065.37.17.453.291.04.06.062.126.059.192-.014.124-.12.229-.28.303a1.242 1.242 0 0 1-.359.092 1.6 1.6 0 0 1-.287.013c-.492-.042-.871-.282-.857-.549a.287.287 0 0 1 .074-.162.425.425 0 0 1 .043-.043c.024-.022.057-.041.088-.06a1 1 0 0 1 .334-.126l.007-.002a1.49 1.49 0 0 1 .381-.015zm2.516 1.487c-.166.062-.46.143-.668.201 1.749 1.036 2.984 2.844 3.2 4.972h.007l.006.131c.006.072.022.14.025.212.014.126.014.252.014.392 0 .022-.004.043-.004.065.08 2.26-.298 3.507-1.309 5.613v.002c-.988 2.091-3.234 4.628-5.007 6.088 6.245-.157 11.277-5.186 11.447-11.445-1.205-1.864-2.381-3.037-3.654-4.024-1.43-1.117-2.725-1.7-4.057-2.207zm2.103 8.254c-.045.119-.086.24-.138.355-.028.057-.056.126-.084.182a6.537 6.537 0 0 1-1.461 1.927l-.295.252c-.14.113-.295.226-.45.338a5.127 5.127 0 0 1-.548.323 6.6 6.6 0 0 1-1.084.457c-2.017.713-5.155.83-8.074.217-1.629-.342-3.145-1.019-4.256-2.128A5.899 5.899 0 0 1 .54 14.812c1.156 5.168 5.69 9.04 11.168 9.178 1.757-1.355 4.24-4.09 5.25-6.229v-.002c.604-1.258.967-2.173 1.148-3.181z"/></svg>
```

Comment:
I'm seeing some small differences between PR (red) and source (black), while the segments are spot on - the path is slightly out of shape. Could be optimization, but would you mind taking a look?

![image](https://user-images.githubusercontent.com/1830867/103171513-d72da180-4844-11eb-94b2-023fa1a05c9f.png)

Reply:
@adamrusted Thanks for pointing this out! I gave it another try and it should be a lot better now.

Note: It might not fit the source up to 100%. Because the gap at the bottom is "cutting off" the edge at the bottom slightly. So the icon was 23,988 after cutting out the gaps. So I had to resize it again to 24px.

Reply:
Is it worth holding off on this until #4530 reaches a conclusion?

Reply:
@adamrusted Sure 👍🏼

Reply:
@adamrusted Can we merge this one now?

Reply:
Certainly can! 🎉